File: c#/summing.cs
using System;

namespace ConsoleApp21
{
    class Program
    {
        static void Main(string[] args)
        {
            int sum = 0;
            int i = 1; 

            while (true)
            {
                Console.WriteLine("Enter a number: ");
                string number = Console.ReadLine(); 

                if (number == "")
                {
                    break; 
                }
                sum += Convert.ToInt32(number);
                i += 1;
                Console.WriteLine(sum);
            }
        }
    }
}

File: c++/summing.cpp
#include "pch.h"
#include <iostream>

using namespace std; 

int main()
{
	int sum = 0;
	int i = 1;

	while (true) {
		cout << "Enter a number: "; 
		int number; 
		cin >> number; 
		if (number == 0)
		{
			break;
		}
		sum += number; 
		i += 1;
		cout << sum << "\n"; 
	}
}


